src/index.html

```diff
@@ -71,7 +76,7 @@
 <body>
   <anms-root>
     <div class="app-loading">
-      <div class="logo"></div>
+      <img src="./assets/logo.png">
```

Comment:
This will not work for the prod build, that was the reason to extract this to css because that way it is processed by the Weback as an asset because the `styles-app-loading.css` is referenced in the `angular.json` as a external stylesheet. Please revert this change.

Reply:
@Jeykairis hmm, looks like this was broken when migrating to Angular CLI 6+ with new `angular.json` format because as it seems the `styles-app-loading.scss` is not referenced anywhere in the whole application... The correct way would be to add it to the `styles` in the `angualr.json`. Another thing worth trying would be to just add the `logo.png` into the `assets` in `angular.json` but then it is the question what would happen during the prod build (eg you don't get the content hashing and cache busting) so maybe it is still better to stick to the `styles-app-loading.scss` which generates hashed asset (eg `logo-24rfsdgd23r.png`).

Reply:
Okay, thank you for the explanations!
I tried to import the `styles-app-loading.scss` into the `styles.scss` and the logo appears but not the spinner...
I try to add the `styles-app-loading.scss` directly in the `angular.json`.

Reply:
@Jeykairis the way this behaves is that the spinner should be shown immediately because it is the part of initial `index.html` template, after that when the styles bundle is loaded for `styles-app-loading.scss` the logo should appear and after that when the whole application is loaded it will replace the spinner and the logo with the app itself ;)